### PR TITLE
Add FTP driver

### PR DESCRIPTION
**Pull request by Xstoudi**

### CODE REVIEW

File: tests/stubs/config.js [hunk 1]
```diff
@@ -17,13 +17,13 @@ module.exports = {
    |
    |
    */
-  'default': 'local',
+  'default': 'ftp',
```
Comment (RomainLanz) [inline tests/stubs/config.js]: This shouldn't change

File: src/Drivers/FTP.js [hunk 2]
```diff
@@ -0,0 +1,200 @@
+const path = require('path')
+const JSFtp = require('jsftp')
```
Comment (RomainLanz) [inline src/Drivers/FTP.js]: I believe this call need to be inside the constructor.

Example: https://github.com/Slynova-Org/node-flydrive/blob/develop/src/Drivers/AwsS3.js#L18

File: src/Drivers/FTP.js [hunk 3]
```diff
@@ -0,0 +1,200 @@
+const path = require('path')
```
Comment (RomainLanz) [inline src/Drivers/FTP.js]: This should be bellow the file header

File: tests/stubs/config.js [hunk 4]
```diff
@@ -38,6 +38,14 @@ module.exports = {
...
       'region': 'AWS_S3_REGION',
       'bucket': 'AWS_S3_BUCKET'
+    },
+
+    'ftp': {
+      'driver': 'ftp',
+      'host': 'FTP_HOST',
+      'port': 21,
+      'user': 'FTP_USER',
+      'pass': 'FTP_PASS'
```
Comment (RomainLanz) [inline tests/stubs/config.js]: Could be great to add an `longLive` that keep the connection open if we have many actions to do and to close it by default after one action.

File: tests/stubs/config.js (hunk 1 above)
Comment (Xstoudi) [reply tests/stubs/config.js]: Fail, forgot to remove this line after dev :)

**Comment by Xstoudi** [comment]
:notes: Somewhere, over the rainbow :notes:

**Pull request closed without merging**